lib/class-wp-block-type.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Blocks API: WP_Block_Type class
+ *
+ * @package gutenberg
+ * @since 0.4.0
+ */
+
+/**
+ * Core class representing a block type.
+ *
+ * @since 0.4.0
+ *
+ * @see register_block_type()
+ */
+class WP_Block_Type {
+	/**
+	 * Block type key.
+	 *
+	 * @since 0.4.0
+	 * @access public
+	 * @var string
+	 */
+	public $name;
+
+	/**
+	 * Block type render callback.
+	 *
+	 * @since 0.4.0
+	 * @access public
+	 * @var callable
+	 */
+	public $render;
```

Comment:
This should perhaps be called `render_callback` to help disambiguate from `render` the method. This is similar to what `WP_Customize_Control` does with its `active` method and `active_callback` member variable.

See https://github.com/WordPress/wordpress-develop/blob/a16d16449ddee6a3c13a65fd02536a6071c04d70/src/wp-includes/class-wp-customize-control.php#L271-L273